win32ss/gdi/ntgdi/freetype.c

```diff
@@ -7166,8 +7183,8 @@ IntExtTextOutW(
 
         previous = glyph_index;
     }
-
-    if (pdcattr->flTextAlign & TA_UPDATECP)
+    /* Don't update position if String == NULL. Fixes CORE-19721. */
```

Comment:
Is `CORE-...` required as code comment, at both places?
```suggestion
    /* Don't update position if String == NULL. */
```

Reply:
@SergeGautherie, Note that in once case we are talking about CORE-11787 and here we are talking about CORE-19721.
I wanted to make it obvious that there are two distinct fixes here. Thanks.